frontend/src/Home.tsx

```diff
@@ -47,19 +81,26 @@ function Home() {
     console.log ("Please select a location");
     } else {
       await BuskApi.createEvent(eventDetails);
+      setEvents(previousData => ([
+        ...previousData,
+        eventDetails
+      ]))
```

Comment:
Make this into a single line instead? e.g. `setEvents(prevData => [...prevData, eventDetails])`